sse2neon.h

```diff
@@ -3568,6 +3581,16 @@ FORCE_INLINE __m128i _mm_minpos_epu16(__m128i a)
             vsetq_lane_s8((b), vreinterpretq_s8_m128i(a), (imm))); \
     })
 
+// Inserts the least significant 8 bits of b into the selected 8-bit integer
+// of a.
+// FORCE_INLINE __m128 _mm_insert_ps(__m128 a, int b,
+//                                   __constrange(0,16) int imm)
+#define _mm_insert_ps(a, b, imm)                                  \
+    __extension__({                                               \
+        vreinterpretq_m128_s8(                                    \
+            vsetq_lane_s8((b), vreinterpretq_s8_m128(a), (imm))); \
+    })
+
```

Comment:
The behaviour of `_mm_insert_ps` is more complicated than `vsetq_lane_s8`.
Please check: [_mm_insert_ps](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_insert_ps&expand=5265,5477,2435,5778,3148)

The implementation should be rewritten or left to the future pull request.

Reply:
Agree, done